### ISSUE TITLE
Does this work?

### ISSUE THREAD

**Issue by bsabiston** [jollymonsterstudio/ue4-kopiarka#2]
I followed the instructions and filled out the application.properties file, but when I run the .jar, it gets an error: 

 ERROR : Source directory does not exist, check your folder configuration

**Comment by CasulScrub** [comment]
Having the same issue.

**Comment by bsabiston** [comment]
The source lists this video as a reference:  I followed it and it worked for me: https://www.youtube.com/watch?v=cmK-A8Dh_Ms

**Comment by MmAaXx500** [comment]
Yes, it works! Use double backslashes like that: D:\\\\UEProj\\\\MyProj
